pkg/asset/manifests/cloudproviderconfig.go

```diff
@@ -91,7 +91,8 @@ func (cpc *CloudProviderConfig) Generate(dependencies asset.Parents) error {
 			return errors.Wrap(err, "failed to get cloud config for openstack")
 		}
 
-		cm.Data[cloudProviderConfigDataKey] = openstackmanifests.CloudProviderConfig(cloud.CloudConfig)
+		cm.Data[cloudProviderConfigDataKey] = openstackmanifests.CloudProviderConfig(cloud.CloudConfig, installConfig.Config.AdditionalTrustBundle != "")
+		cm.Data["additionalTrustBundle"] = installConfig.Config.AdditionalTrustBundle
```

Comment:
By adding the `additionalTrustBundle` to the `cloud-provider-config` configmap, you're making it dfficult to mount the file to `/etc/pki/openshift-ca-trust/additionalTrustBundle.pem`

The cert is already stored in the `openshift-config/user-ca-bundle` config map.

Reply:
Hmm.. this is what @deads2k outlined as the way to add the CA cert for cloud-provider. Is the openshift-config configmap mounted to cloud provider services?

Reply:
Ok, then it means we need to update our in flight patches to read from `openshift-config/cloud-provider-config` rather than `openshift-config/user-ca-bundle`.

BTW, it's possible to mount some files from a configmap to a different path, so it's not that big of a deal that we're putting the CA cert in the cloud-provider-config configmap.

https://kubernetes.io/docs/concepts/storage/volumes/#configmap